src/deploy.sh

```diff
@@ -476,6 +478,11 @@ function kernel_deploy()
     fi
   fi
 
+  if [ -n "$pkg" ]; then
+    pkg_deploy "" "$pkg" "$target" "$remote"
```

Comment:
'' instead of ""

Reply:
Notice that you are skipping the statistic data. It would be nice if you also consider keeping track of the build time using pkg and differentiating it from the normal build, enabling us to compare performance.
@magalilemes might be interested in this part.

Reply:
Hmmm... I also think that you should keep this part close to the kernel_install code. Did you check if we can only build the Debian package for modules? I doubt we have this option, but I think it is worth checking.